packages/portal-proto/src/features/cohortBuilder/QueryExpressionSection.tsx

```diff
@@ -230,13 +206,11 @@ const QueryExpressionSection: React.FC<QueryExpressionSectionProps> = ({
 
                 <Tooltip
                   label={
-                    filtersSectionCollapsed
-                      ? numOfRows <= MAX_COLLAPSED_ROWS
-                        ? "All rows are already displayed"
-                        : "Display all rows"
-                      : numOfRows <= MAX_COLLAPSED_ROWS
-                      ? `A maximum of ${MAX_COLLAPSED_ROWS} rows is already displayed`
-                      : `Display a maximum of ${MAX_COLLAPSED_ROWS} rows`
+                    noFilters
+                      ? "No filters to show/hide"
+                      : filtersSectionCollapsed
+                      ? "Show all filters"
+                      : "Hide some filters"
```

Comment:
i think more like this, but i may tweak the text a little bit more later. The "100" is just temporary.

```
                  label={
                    noFilters || QESectionHeight <= 100 
                    ? "All rows are already displayed"
                    : filtersSectionCollapsed 
                      ? "Display all rows"
                      : "Display fewer rows"
                  }
```